common/models/update_environment_request.go

```diff
@@ -7,8 +7,9 @@ import (
 )
 
 type UpdateEnvironmentRequest struct {
-	EnvironmentID   string `json:"environment_id"`
-	MinClusterCount *int   `json:"min_cluster_count"`
+	EnvironmentID   string    `json:"environment_id"`
```

Comment:
Need to remove `EnvironmentID` field here. We will need a new object to pass into the job runner.

As a pattern maybe we swap the `-Request` suffix with `-Job` and wrap the original request?

```
type UpdateEnvironmentJob struct {
    EnvironmentID string
    Req           UpdateEnvironmentRequest
}
```

Reply:
See above.